src/py21cmfast/src/IonisationBox.c

```diff
@@ -105,6 +105,8 @@ LOG_SUPER_DEBUG("initing heat");
     init_heat();
     float TK;
     TK = T_RECFAST(redshift,0);
+    float cT_ad;
```

Comment:
Can you write a comment on top of this that briefly says what you're calculating here?

Reply:
is cT_ad a float or a double? I think we define the function as a double